cmake/Common.cmake

```diff
@@ -4,10 +4,10 @@ if (NOT MK_ROOT)
     set (MK_ROOT ${CMAKE_SOURCE_DIR})
 endif (NOT MK_ROOT)
 
-set(CMAKE_BUILD_TYPE Debug CACHE STRING "Set the build type")
+set(CMAKE_BUILD_TYPE Release CACHE STRING "Set the build type")
```

Comment:
I changed the build type because our dependencies are using `Release` only, for now, and not doing that leads us to library mismatches when we're linking.